components/mosart/src/riverroute/RunoffMod.F90

```diff
@@ -251,6 +255,8 @@ module RunoffMod
      integer , pointer :: iUp(:,:)     ! IDs of upstream units, corresponding to the subbasin ID in the input table
```

Comment:
A few variables such as `iUp` `nUp` are now defined in `rtmCTL`. Consider removing them from `TUnit`.

Reply:
Done.